backend/algorithms/objects/categories.py

```diff
@@ -49,6 +51,12 @@ def set_logic(self, logic: Logic):
         """ AND or OR """
         self.logic = logic
 
+    def is_child(self, child: bool):
+        self.child = child
+    
+    def set_isWrapAnd(self, isWrap: bool):
```

Comment:
i think pylint will complain abt this not being snake case

Reply:
ah yes will add to my list of fixing for tomorrow '.'